webconsole/src/main/java/com/pavlovmedia/oss/jaxrs/webconsole/JaxrsConsole.java

```diff
@@ -90,6 +98,7 @@ protected void renderContent(final HttpServletRequest req, final HttpServletResp
     }
     
     private void renderProviderSet(final PrintWriter pw) {
+        logger.info("Inside JaxrsConsole's renderProviderSet");
```

Comment:
All of these infos are going to make a log of noise in the log, are they needed now?

Reply:
They were added solely for the testing purpose. They're not needed anymore. I will take those off.